### PR TITLE
Support feature with one or more tensors for Sample in python 

### PR DESCRIPTION
**Pull request by hkvision**
## What changes were proposed in this pull request?

1. Support one or more `Tensor` (list of tensors) for feature in `Sample` on Python side.
2. Add test for the change. Model having multiple tensor as inputs.
3. Correct minor typo in docs.

## How was this patch tested?

Jenkins.

### CODE REVIEW

**Comment by hkvision** [comment]
pr validation passed http://172.168.2.101:8080/view/PR-Validation/job/BigDL-PR-Validation/2035/

**Comment by hkvision** [comment]
Correct typo in docs https://bigdl-project.github.io/master/#ProgrammingGuide/Model/Sequential/#define-a-model-with-multiple-inputs for CAddTable().

**Comment by hkvision** [comment]
pr validation after adding test passed: http://172.168.2.101:8080/view/PR-Validation/job/BigDL-PR-Validation/2045/

File: pyspark/bigdl/util/common.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ class Sample(object):
     def __init__(self, features, label, bigdl_type="float"):
         """
         User should always use Sample.from_ndarray to construct Sample.
-        :param features: a JTensor
+        :param features: a JTensor or a list of JTensors
```
Comment (jason-dai) [inline pyspark/bigdl/util/common.py]: feature is now a list of JTensors, no?

**Comment by hkvision** [comment]
After removing toPySample, pr validation still passed: http://172.168.2.101:8080/view/PR-Validation/job/BigDL-PR-Validation/2050/
Seems toPySample is only used for unit test?

File: pyspark/bigdl/util/common.py (hunk 1 above)
Comment (hkvision) [reply pyspark/bigdl/util/common.py]: Corrected.

**Comment by hkvision** [comment]
Recover toPySample. It can be used only to validate feature having one tensor.

File: pyspark/bigdl/util/common.py [hunk 2]
```diff
@@ -179,7 +179,7 @@ class Sample(object):
     def __init__(self, features, label, bigdl_type="float"):
         """
         User should always use Sample.from_ndarray to construct Sample.
-        :param features: a JTensor
+        :param features: a list of JTensors
         :param label: a JTensor
```
Comment (qiuxin2012) [inline pyspark/bigdl/util/common.py]: labels is also a list of JTensors

Comment (hkvision) [reply pyspark/bigdl/util/common.py]: Keep label as one JTensor for now. May need update in the future.

**Comment by hkvision** [comment]
http://172.168.2.101:8080/view/PR-Validation/job/BigDL-PR-Validation/2060/

**Pull request merged**